### DISCUSSION TITLE
SSR Support

### DISCUSSION THREAD

**Discussion by hasib-devs** [akiran/react-slick#2303] [General]
Hey React Slick community!
I hope everyone is doing well. I've been working on integrating React Slick into my NextJS project, and I'm curious about its SSR (Server-Side Rendering) support. I've scoured the documentation and issues, but I couldn't find a definitive answer.
Can anyone shed some light on whether React Slick officially supports SSR? If so, are there any best practices or considerations I should be aware of when using it in an SSR environment?

I've seen some discussions around this topic, but it would be great to have a consolidated thread where we can share experiences, tips, and possibly get insights from the maintainers.

Looking forward to hearing from the community and getting some clarity on this matter! Thanks in advance for your help. 🚀

**Comment by hassam-saeed** [comment]
I am also looking for an SSR solution.
